runtime/docker-entrypoint.d/02-install-extension

```diff
@@ -8,6 +10,8 @@ run_as() {
     fi
 }
 
+run_as "php occ config:system:set --type=\"integer\" --value=\"${LOG_LEVEL}\" \"loglevel\""
+
```

Comment:
This is not a desired config in production.

Reply:
Ok let's keep it only in readme so.